src/components/Navigation/Navigation.tsx

```diff
@@ -29,6 +29,12 @@ export const Navigation = (): ReactElement => {
         >
           Staff
         </NavLink>
+        <NavLink
+          className={({ isActive }) => cn('nav__item', { 'nav__item--active': isActive })}
+          to={RouteConstants.CHECKOUT}
+        >
+          Checkout
+        </NavLink>
```

Comment:
Do we want this in the nav bar? I feel like it should live on the order summary on the right of the screen?

Reply:
I see that it already does. I just don't think we need it on the nav bar as well